packages/radix-vue/package.json

```diff
@@ -57,17 +57,12 @@
     "pub:release": "pnpm publish --access public"
   },
   "peerDependencies": {
-    "@internationalized/date": "^3.5.2",
     "vue": ">= 3.2.0"
   },
-  "peerDependenciesMeta": {
-    "@internationalized/date": {
-      "optional": true
-    }
-  },
   "dependencies": {
     "@floating-ui/dom": "^1.5.4",
     "@floating-ui/vue": "^1.0.4",
+    "@internationalized/date": "^3.5.2",
```

Comment:
I saw @epr3 updated to use `"@internationalized/date": "^3.5.1"` in devDependencies, perhaps we should use the same here?

Reply:
Do you want to change it to 3.5.1? We don't need it to be dev dependency now. What do you suggest? 3.5.2 is the latest version, should be fine

Reply:
Yes, use the latest version.  👍

Reply:
Cool! Just wanna make sure the version is the same as in the dev version.